20201022 release (#767)
publiccodenet/about: activities/tool-management/tentoo.md

NMBRS is a fully-fledged HR service and registers employee's contact and personal details, such as date of birth, home address and who their emergency contacts are. It also has details of contract terms, a leave overview and several mutation forms needed for changes in contract terms, salary or home address. Tentoo's HR department processes the information and is responsible for producing the payslips and other reports pertaining to salaries and taxes, such as income taxes. The Operations coordinator transfers the salaries by uploading the SEPA file to the bank. The Chief executive releases the payments.

## Tentoo privacy policy

> Tentoo's privacy policy will be published on their newly designed website shortly. As soon as it's live we will link to it from here.

Reviewer: I don't think this placeholders are ever a good solution. With this formulation, we're taking responsibility for their actions. I suggest:

```suggestion
Tentoo's privacy policy is not published. They have assured us it will be soon.
```

Author: This has been superseded by #772.
